docs/fsharp/language-reference/members/methods.md

```diff
@@ -96,6 +100,18 @@ The following example illustrates a derived class that overrides a base class me
 ## Overloaded Methods
 Overloaded methods are methods that have identical names in a given type but that have different arguments. In F#, optional arguments are usually used instead of overloaded methods. However, overloaded methods are permitted in the language, provided that the arguments are in tuple form, not curried form.
 
+## Optional Arguments
+
+Starting with F# 4.1, you can also have optional arguments in methods.  You can also specify a default parameter value.  This is to help facilitate interoperation with C# and Visual Basic .NET code.  The following example demonstrates the syntax:
```

Comment:
Optional arguments have been supported since F# 2.0 and have their own section https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/parameters-and-arguments

What's been added in F# 4.1 is the ability to make C#-style optional arguments with a default parameter value.